### ISSUE TITLE
0.20 regression: mapping to `./` generates two slashes in path `//`

### ISSUE THREAD

**Issue by christopherthielen** [systemjs/systemjs#1609]
Please see this example which is using 0.19:
https://plnkr.co/edit/V7WidRpNvgqHoCS7hHbl?p=preview

Then the same example using 0.20.9:
https://plnkr.co/edit/p7LKWzDX06QZ66ZCpaPA?p=preview

Here is our config:

```js
System.config({
  map: {
    "react": "//unpkg.com/react@15/dist/react.js",
    "react-dom": "//unpkg.com/react-dom/dist/react-dom.js",
    "ui-router-react": "//unpkg.com/ui-router-react@0.4.0/_bundles/ui-router-react.js",
    "helloworld": "./"
  },
  packages: {
    helloworld: {
      main: './helloworld.js',
      defaultExtension: 'js'
    }
  }
})
```

With 0.19, this fetches a URL such as `https://run.plnkr.co/LBETNubaB40KIAtj/helloworld.js`.

With 0.20, this fetches `https://run.plnkr.co/LBETNubaB40KIAtj//helloworld.js`

I haven't been able to come up with a configuration for 0.20 which enables the same "./" mapping.  Helloworld is a simple app with only one source file, but we have more complex tutorials also which load multiple files (which is why I believe the "map" to "./" is necessary): https://plnkr.co/edit/3qSxCVrm1KuDTVH9nHjO?p=preview

**Comment by guybedford** [comment]
I've added a fix in https://github.com/systemjs/systemjs/commit/720aa9e4e5b4051f8c05ea2cf5ce2bf702813679. Thanks for the clear test case here.

**Comment by guybedford** [comment]
Released in 0.20.10.
